src/components/TechnicalInfoContent/TechnicalInfoContent.js

```diff
@@ -0,0 +1,93 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+
+import React from 'react';
+import { Button, Grid, List, ListItem, Typography } from '@mui/material';
+import ConfigService from '../../services/ConfigService';
+import { useSolutionData } from '../../state/hooks/SolutionHooks';
+import { useWorkspaceData } from '../../state/hooks/WorkspaceHooks';
+import { useTranslation } from 'react-i18next';
+
+export const TechnicalInfoContent = () => {
+  const { t } = useTranslation();
+  const solutionData = useSolutionData();
+  const solutionName = solutionData?.name ?? '';
+  const solutionDescription = solutionData?.description ?? '';
+  const currentWorkspace = useWorkspaceData();
+
+  const technicalInformation = [
+    {
+      id: 'webappVersion',
+      label: t('genericcomponent.dialog.technicalInfo.webAppVersion', 'Webapp version'),
+      content: ConfigService.getParameterValue('APP_VERSION'),
+    },
+    {
+      id: 'registryName',
+      label: t('genericcomponent.dialog.technicalInfo.repositoryName', 'Container registry repository name'),
+      content: solutionData?.repository,
+    },
+    {
+      id: 'registryTag',
+      label: t('genericcomponent.dialog.technicalInfo.repositoryTag', 'Container registry repository tag'),
+      content: solutionData?.version,
+    },
+    {
+      id: 'SDKVersion',
+      label: t('genericcomponent.dialog.technicalInfo.sdkVersion', 'SDK version'),
+      content: solutionData?.sdkVersion,
+    },
+  ];
+
+  const infoToDisplay = currentWorkspace
+    ? technicalInformation
+    : technicalInformation.filter((infoItem) => ['webappVersion'].includes(infoItem.id));
+
+  const copyInfoToClipboard = () => {
+    const notSpecifyEntry = t('genericcomponent.dialog.technicalInfo.notSpecifiedEntry');
+    const infoToDisplayList = infoToDisplay
+      .map((infoItem) => `${infoItem.label} ` + (infoItem.content ?? `${notSpecifyEntry} (${infoItem.content}`))
```

Comment:
Closing parenthesis is missing in the string built when `infoItem.content` is not defined